bls12_381/Cargo.toml

```diff
@@ -20,6 +20,15 @@ name = "groups"
 harness = false
 required-features = ["groups"]
 
+[dependencies.ff]
```

Comment:
I kinda want `ff` /  `group` / `pairing` to be optional dependencies, somehow. I figured `pairing` would be brought in by the `pairings` feature, and that `group` would be brought in by the `groups` feature. I hadn't thought about `ff` though. Hm..

Reply:
`fields` feature? 😛

Reply:
(and then have `groups =  ["fields"]` which would make it available for both `groups` and `pairings`)

Reply:
That sounds like a good approach, but it means: `fields` enables `Field` but the scalar field implementation is exported anyway, whereas `groups` enables both the `group` APIs and the implementations as well. This is a strange asymmetry.

Reply:
Ah, well then `ff` should be a required dependency for the field implementation, given that `group` is a required dependency when using the group implementation.

Reply:
Alright, so we'll make ff a required dependency. You can consider this point resolved.